Update Readme.md (#1)
DevExpress-Examples/winforms-chart-create-and-customize-a-constant-line: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1370)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# Chart for WinForms - How to Create and Customize a Constant Line

This example demonstrates how create and customize a [constant line](https://docs.devexpress.com/WindowsForms/5783/controls-and-libraries/chart-control/axes/constant-lines) at runtime.

![Constant Line](image/chart.png)

> Since constant lines belong to a diagram's [axis](https://docs.devexpress.com/WindowsForms/5779/controls-and-libraries/chart-control/diagram/axes), it is required to cast the [diagram](https://docs.devexpress.com/WindowsForms/5778/controls-and-libraries/chart-control/diagram) object to the appropriate diagram's type. After that, you can access axes and manage their constant lines collection.

Reviewer: ```suggestion
Since constant lines belong to a diagram's [axis](https://docs.devexpress.com/WindowsForms/5779/controls-and-libraries/chart-control/diagram/axes), you need to cast the [diagram](https://docs.devexpress.com/WindowsForms/5778/controls-and-libraries/chart-control/diagram) object to the appropriate diagram's type. After that, you can access axes and manage their constant lines collection.
```

Why 'appropriate'? And what does `the appropriate diagram's type` mean?

> Since constant lines belong to a diagram's [axis](https://docs.devexpress.com/WindowsForms/5779/controls-and-libraries/chart-control/diagram/axes), it is required to cast the [diagram](https://docs.devexpress.com/WindowsForms/5778/controls-and-libraries/chart-control/diagram) object to the appropriate diagram's type. After that, you can access axes and manage their constant lines collection.

Author: Since constant lines belong to a diagram's [axis](https://docs.devexpress.com/WindowsForms/5779/controls-and-libraries/chart-control/diagram/axes), you need to cast the [diagram](https://docs.devexpress.com/WindowsForms/5778/controls-and-libraries/chart-control/diagram) object to the appropriate diagram's type. After that, you can access axes and manage their constant lines collection.

> Since constant lines belong to a diagram's [axis](https://docs.devexpress.com/WindowsForms/5779/controls-and-libraries/chart-control/diagram/axes), it is required to cast the [diagram](https://docs.devexpress.com/WindowsForms/5778/controls-and-libraries/chart-control/diagram) object to the appropriate diagram's type. After that, you can access axes and manage their constant lines collection.

Author: ```suggestion
Since constant lines belong to a diagram's [axis](https://docs.devexpress.com/WindowsForms/5779/controls-and-libraries/chart-control/diagram/axes), you need to cast the [diagram](https://docs.devexpress.com/WindowsForms/5778/controls-and-libraries/chart-control/diagram) object to the appropriate diagram's type. After that, you can access axes and manage their constant lines collection.
```
